src/views/tickets/orders.php

```diff
@@ -6,12 +6,13 @@
  *
  * @package TribeEventsCalendar
  *
- * @since 4.7.4
- * @since 4.10.2 Only show Update button if ticket has meta.
- * @since 4.10.8 Show Update button if current user has either RSVP or Ticket with meta. Do not use the now-deprecated third parameter of `get_description_rsvp_ticket()`.
- * @since 4.10.9 Use function for text.
+ * @since   4.7.4
+ * @since   4.10.2 Only show Update button if ticket has meta.
+ * @since   4.10.8 Show Update button if current user has either RSVP or Ticket with meta. Do not use the now-deprecated third parameter of `get_description_rsvp_ticket()`.
+ * @since   4.10.9 Use function for text.
+ * @since   TBD Correct getting `$event_id` when using The Events Calendar's "Default Page Template" display template.
```

Comment:
```suggestion
 * @since   TBD Correct getting `$event_id` when using The Events Calendar's "Default Page Template" display template. `$event_id` now relies on the `WP_Query` queried object ID instead of the global `$post` object.
```